spec/requests/api/v1/metrics_spec.rb

```diff
@@ -92,20 +92,20 @@
     end
 
     it 'returns `feedex issues` between two dates' do
-      get "/api/v1/metrics/number_of_issues/#{content_id}/time-series", params: { from: '2018-01-13', to: '2018-01-15' }
+      get "/api/v1/metrics/feedex_comments/#{content_id}/time-series", params: { from: '2018-01-13', to: '2018-01-15' }
```

Comment:
In this case we are changing an API endpoint that we shouldn't change (we are changing the URL of an API of another system)

Reply:
Sorry, @KyleMacPherson my bad. This is ok, as it is our internal API.
Discard this message, please.